tests/unit_tests/blas/extensions/imatcopy_usm.cpp

```diff
@@ -151,6 +151,10 @@ int test(device *dev, oneapi::mkl::layout layout) {
         print_error_code(e);
     }
 
+    catch (const oneapi::mkl::unimplemented &e) {
```

Comment:
A similar catch should probably be added to the buffer tests, I will look at it.

Reply:
Never mind, it's already there (as you probably knew).